modules/cluster/irsa.tf

```diff
@@ -397,9 +397,9 @@ data "aws_iam_policy_document" "secrets-manager-policy" {
       "secretsmanager:UpdateSecret",
     ]
     resources = [
-      "arn:${data.aws_partition.current.partition}:secretsmanager:${var.region}:${local.project}:secret:secret/data/lighthouse/*",
-      "arn:${data.aws_partition.current.partition}:secretsmanager:${var.region}:${local.project}:secret:secret/data/jx/*",
-      "arn:${data.aws_partition.current.partition}:secretsmanager:${var.region}:${local.project}:secret:secret/data/nexus/*"
+
```

Comment:
We could make it a variable, and default to the three resources, if not resource list is provided. Let's stick to Principle of least privilege. Also, imo, this may not be required at all, See my comment: https://github.com/jenkins-x/terraform-aws-eks-jx/issues/294#issuecomment-903413806